fiftyone/utils/torch.py

```diff
@@ -114,6 +114,12 @@ class TorchImageModelConfig(foc.Config):
         image_min_dim (None): resize input images during preprocessing, if
             necessary, so that the smaller image dimension is at least this
             value
+        image_max_size (None): resize the input images during preprocessing, if
+            necessary to prevent OOM, so that the image dimensions are at 
```

Comment:
nit: I'd recommend removing the "to prevent OOM" commentary from the docstrings in this PR. It's one reason why this parameter might be used, but there could be other reasons